lti_consumer/tests/unit/test_models.py

```diff
@@ -90,6 +113,7 @@ def test_lti_consumer_ags_enabled(self):
 
         # Get LTI 1.3 consumer
         consumer = self.lti_1p3_config.get_lti_consumer()
+        self.maxDiff = None
 
         # Check that LTI claim was included in extra claims
         self.assertEqual(
```

Comment:
You need to update LTI configuration id to ``3`` for the LTI AGS test [here](https://github.com/open-craft/xblock-lti-consumer/blob/kshitij/lti-on-model-config-lti1.1/lti_consumer/tests/unit/test_models.py#L129). Because your change on this PR creates a new LTI configuration and ``lti_1p3_config`` is the last one. I think this is why the test is failing.

Reply:
Done.